z_calibration.py

```diff
@@ -137,6 +132,17 @@ def cmd_CALIBRATE_Z(self, gcmd):
         if self.state is not None:
             raise self.printer.command_error("Already performing CALIBRATE_Z")
             return
+        if self.config.getfloat('probe_bed_x', None) is None \
+            or self.config.getfloat('probe_bed_y', None) is None:
+            try:
+                mesh = self.printer.lookup_object('bed_mesh', default=None)
```

Comment:
Here is the code for checking for parameters. I tested that this also works on my printer:
```
probe_bed_x = gcmd.get_float('PROBE_BED_X', default = None)
probe_bed_y = gcmd.get_float('PROBE_BED_Y', default = None)
if probe_bed_x is not None and probe_bed_y is not None:
    self.probe_bed_site = [probe_bed_x, probe_bed_y]
else:
    mesh = self.printer.lookup_object('bed_mesh', default=None)
    rri = mesh.bmc.relative_reference_index    
    self.probe_bed_site = mesh.bmc.points[rri]
    logging.debug("Z-CALIBRATION probe_bed_x=%.3f probe_bed_y=%.3f"
            % (self.probe_bed_site[0], self.probe_bed_site[1]))
```

Reply:
Maybe I can add this as an additional option you can use, when not using a mesh. People then can define the center of the print as probe point.